<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DOE Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Y Responce</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="YcomboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Interactions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="interactionlistWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>105</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Ylabel">
          <property name="text">
           <string>Interaction Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="selectedlistWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="includeallcheckBox">
            <property name="text">
             <string>Include All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>Plot</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="coefradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Coefficient</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="fitradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Fitting</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="coefpredradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>CV Coefficient</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="predradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>CV Prediction</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="dismradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Dispersion Matrix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="inflradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Iflaction Factor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="levradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Leverage</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="surradioButton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Surface</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="ApplyButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Apply</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="MarkergroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Marker</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QCheckBox" name="XcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>X Label</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="XGcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>X Grid</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="YcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Y Label</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="YGcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Y Grid</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="YMcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Y Marks</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="XMcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>X Marks</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="VcheckBox">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Values</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Labelling</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Title</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="XlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Y axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="YlineEdit">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="mplwindow" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="mplvl"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>698</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>centralwidget</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
